frontend/packages/dev-console/src/components/pipelines/pipeline-builder/task-sidebar/Sidebar.tsx

```diff
@@ -6,7 +6,7 @@ import './Sidebar.scss';
 type LazyRender = () => React.ReactNode;
 type SidebarProps = {
   children: React.ReactNode | LazyRender;
-  onRequestClose: () => void;
+  onRequestClose: (e: Node) => void;
```

Comment:
Change no longer needed it seems.

Reply:
:/ I was going to keep it so the calling code could manage any additional overlaps... but I can remove it.